CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## VERSION 2.1.0
+_04_10_2021_
```

Comment:
_04_11_2021_ we are in november already, almost christmas :P

Reply:
listo